opm/parser/eclipse/EclipseState/Tables/PlyshlogTable.hpp

```diff
@@ -102,19 +96,21 @@ namespace Opm {
             return m_hasRefTemperature;
         }
 
-        void setHasRefSalinity(const bool has){
+        void setHasRefSalinity(const bool has) {
             m_hasRefSalinity = has;
         }
 
-        void setHasRefTemperature(const bool has){
+        void setHasRefTemperature(const bool has) {
             m_refTemperature = has;
         }
 
-        const std::vector<double> &getWaterVelocityColumn() const
-        { return getColumn(0); }
+        const TableColumn& getWaterVelocityColumn() const {
+            return getColumn(0);
```

Comment:
while at this: introducing enums for the column indices would probably be a good thing. (e.g., `getColumn(WATER_VELOCITY_COLUMN)`). that's a bit of work, though.